src/containers/shared/utils.js

```diff
@@ -21,6 +21,29 @@ export const NOT_FOUND = 404
 export const SERVER_ERROR = 500
 export const BAD_REQUEST = 400
 
+export const FETCH_INTERVAL = 5000
```

Comment:
```suggestion
export const FETCH_INTERVAL_MILLIS = 5000
```

It's often helpful to include the units, even if they seem obvious from the way the number is formatted :)